### DISCUSSION TITLE
response 504

### DISCUSSION THREAD

**Discussion by Enricond** [fattureincloud/api#270] [Q&A] [answered]
sto riscontrando problemi nell'accesso alle api.
anche usando il pannello degli esempi.

da qui https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/issued_documents
mettendo i miei dati (token e company) ho questo responso
Response Status: Failed to fetch (CORS or Network Issue)

**Comment by valmoz** [comment, marked as answer]
Buongiorno @Enricond 
il problema sul server dovrebbe essere stato risolto.

Per questo genere di problemi la prossima volta vi consiglio di contattare il supporto clienti.

Grazie
Mauro

**Reply by Enricond** [reply]
sì, risolto grazie.
